src/collections/Grid/GridRow.js

```diff
@@ -53,7 +54,7 @@ GridRow._meta = {
   type: META.TYPES.COLLECTION,
   props: {
     color: SUI.COLORS,
-    columns: SUI.WIDTHS,
+    columns: [...SUI.WIDTHS, 'equal'],
```

Comment:
Here is fix for #599. `Row` can [have](https://github.com/Semantic-Org/UI-Grid/blob/master/grid.css#L1469) `equal width`.